src/detours.cpp

```diff
@@ -1329,6 +1329,12 @@ struct DetourOperation
     ULONG               dwPerm;
 };
 
+struct MiniheapHeader {
```

Comment:
For consistency with the surrounding newline style.
```suggestion
struct MiniheapHeader
{
```